client/js/lounge.js

```diff
@@ -275,7 +286,7 @@ $(function() {
 
 			chat.find(".chan.active .chat").trigger("msg.sticky"); // fix growing
 		})
-		.tab(completeNicks, {hint: false})
+		.tab(completeNicks(false), {hint: false})
```

Comment:
> Your solution: that inner function, called three times with (word) => completeNicks(word, false/true)

It's just not about function calls, it's about being clear what the function does. It is a lot cleaner code when the function just takes 2 arguments instead of 1 argument and return a function that takes a secondary argument (there's literally no reason to do this, we don't do this anywhere else in the entire codebase).